addons/functions/CfgVehicles.hpp

```diff
@@ -138,6 +138,7 @@ class CfgVehicles {
         ace_trenches_removalDuration = QGVAR(smallEnvelopeRemovalTime);
         scopecurator = 2;
         scope = 2;
+        grad_trenches_damageMultiplier = 3;
```

Comment:
Wouldn't it make sense to make this an Option in the CBA Options?

Reply:
you mean for each trench? its meant to control individual "structural strength" of smaller vs larger trenches. e.g. small trenches are destroyed by one HE-shell, larger only partly.

Reply:
Yeah, so that mission builders can choose.

Reply:
soo.. what are you suggesting exactly? making a slider for each trench type? disabling the difference between trenches by checkmark?

Reply:
No, like a Slider, and then the code in the config would look like this:
`ace_trenches_diggingDuration = QGVAR(giantEnvelopeDigTime);`

Reply:
i feel this is a bit too detailed for a setting / cant imagine someone has the need for it and its crowding the already extensive cba settings.

a1fea33ef5a2f3eef8bd7c8120984676bdf672fb